clusters/ithc/base/kustomization.yaml

```diff
@@ -26,9 +26,10 @@ resources:
   - ../../../apps/dg/base/kustomize.yaml
   - ../../../apps/aac/base/kustomize.yaml
   - ../../../apps/am/base/kustomize.yaml
-  # MountVolume.SetUp failed for volume "vault-bar”. A secret with (name/id) bar-IDAM-CLIENT-SECRET was not found in this key vault.
+  # 2023-07-24 MountVolume.SetUp failed for volume "vault-bar”. A secret with (name/id) bar-IDAM-CLIENT-SECRET was not found in this key vault.
   # - ../../../apps/bar/base/kustomize.yaml
-  - ../../../apps/camunda/base/kustomize.yaml
+  # 2023-07-25 camunda-optimise-base: Startup probe failed: Get "http://10.11.208.76:8090/health/liveness": dial tcp 10.11.208.76:8090: connect: connection refused
+  # - ../../../apps/camunda/base/kustomize.yaml
```

Comment:
since this is our app we need to raise a story / bau task to fix it, but i think it doesn't need to block this PR